packages/bpk-component-text/src/BpkText.js

```diff
@@ -26,7 +26,17 @@ import STYLES from './BpkText.scss';
 
 const getClassName = cssModules(STYLES);
 
-const TEXT_STYLES = ['xs', 'sm', 'base', 'lg', 'xl', 'xxl'];
+const TEXT_STYLES = [
+  'xs',
+  'sm',
+  'base',
+  'lg',
+  'xl',
+  'xxl',
+  'xxxl',
+  'xxxxl',
+  'xxxxxl',
```

Comment:
iOS stops at xxxl. Should we and native be aligned?

Reply:
So these are new ones that Will has defined that are to be used for web use cases as part of the the web type ramp but Will is also reviewing about having these new ones on Native platforms as well